docs/design.md

```diff
@@ -104,6 +104,15 @@ These blocks are called _address blocks_, and assigned to nodes.  Since all IP a
 
 For example, if an address pool defines that the size of an address block is 2<sup>5</sup>, `coil-controller` will curve an address block for IPv4 with `/27` subnet mask out of the pool, and assigns it to a node.
 
+When assigning an address, if `coil-controller` assigns the last released address again, it is possible to use old routes and it is not possible to establish a new connection via egress NAT.
+Therefore, avoiding reassigning the last released address, it records the index of the last assigned address.
```

Comment:
I'd say something like, "In general, avoiding immediate reuse of IP addresses is better to not confuse other software or peripherals.".